nursery/coin_select/Cargo.toml

```
@@ -0,0 +1,11 @@
+[package]
+name = "bdk_coin_select"
+version = "0.0.1"
+authors = [ "LLFourn <lloyd.fourn@gmail.com>" ]
```

Even though @LLFourn is the primary contributor on this, for consistency and since other's will be contributing in the future I think best not to have individual names here.
```suggestion
authors = ["Bitcoin Dev Kit Developers"]
```